src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/DeriveHearingCentreHandlerTest.java

```diff
@@ -351,17 +352,98 @@ void should_not_set_hearing_centre_if_already_exists() {
     }
 
     @Test
-    void should_set_default_hearing_centre_for_detained_appeals() {
+    void should_set_default_hearing_centre_for_detained_appeals_ada_or_aaa() {
```

Comment:
Can it be AAA?

Reply:
This test was added as part of a previous ticket, the rename is just to reflect the latest changes which don't apply Harmondsworth to all detained appeals anymore. So for detained ADA or detained AAA it will go to Harmondsworth.